towncrier_template.rst

```diff
@@ -0,0 +1,27 @@
+{% for section, _ in sections.items() %}
+{% set underline = underlines[0] %}{% if section %}{{section}}
+{{ underline * section|length }}{% set underline = underlines[1] %}
```

Comment:
There's only one entry configured for "underlines" in pyproject.toml; should this `set` tag be removed?  `underline` doesn't seem to be referenced after this in the template.

Reply:
Yeah, I did it this way so that we can eventually start adding section headers if we want.

Maybe I should just pull it out and we'll add it in if and only if we need it.